ui/js/actions/wallet.js

```diff
@@ -85,13 +86,13 @@ export function doSendDraftTransaction() {
         dispatch({
           type: types.SEND_TRANSACTION_COMPLETED,
         });
-        dispatch(doOpenModal("transactionSuccessful"));
+        dispatch(doOpenModal(modals.TRANSACTION_SUCCESSFUL));
```

Comment:
Let's drop this modal and replace with a snackbar showing:

You sent ${amount} LBC. [History]

History is a link to the Transaction History page.